_templates/layout.html

```diff
@@ -1,6 +1,8 @@
 {% extends "!layout.html" %}
 
 {%- block extrahead %}
+
+{% if enable_cookies|default(true) %}
```

Comment:
this comes from here: https://github.com/conan-io/docs/pull/3856 but with the filter it should work even if we don't define the variable in the conf.py